data-host/library/src/popup/components/DataScreen/WebpageViews.tsx

```diff
@@ -18,7 +18,7 @@ function HistoryList({ n }: { n: number }) {
   const history = useLoadedState(async () => {
     const items = [];
     for await (const item of getDataHost().iterBack()) {
-      if (item.pageView == null) continue;
+      if (!item || item.pageView == null) continue;
```

Comment:
Did we change iterBack to sometimes return null? We shouldn't do that. The iter methods on data host should only return valid facts.

Reply:
There was a moment in which the facts were wrongly sent. Thats when I added the check and left it there. I can remove it.